gammapy/maps/hpx/ndmap.py

```diff
@@ -451,13 +456,13 @@ def smooth(self, width, kernel="gauss"):
         Parameters
         ----------
         width : `~astropy.units.Quantity`, str or float
-            Smoothing width given as quantity or float. If a float is given it
+            Smoothing width given as quantity or float. If a float is given it is
             interpreted as smoothing width in pixels. If an (angular) quantity
-            is given it converted to pixels using ``healpy.nside2resol``.
+            is given it is converted to pixels using ``healpy.nside2resol``.
```

Comment:
Do you have any idea why ``healpy.nside2resol`` does not render as a link in the doc ? If you try to add `~` before?
My other guess would be that it is because healpy is not in the core dependencies.

Reply:
Yes, I believe it is due to healpy not being a dependent package

Reply:
I am not sure, healpy is among the packages that are requested for docs-build. The syntax is not correct I think

Reply:
Not sure if you tried making the docs with this line, but it does not show as a clickable link for me, it just shows the tilde. 
![image](https://github.com/gammapy/gammapy/assets/60767360/13566e3d-7d90-4e2f-a9ab-0e978e86097d)